kroxylicious-integration-tests/src/test/java/io/kroxylicious/proxy/filter/validation/JsonSchemaRecordValidationIT.java

```diff
@@ -177,7 +183,8 @@ void nonExistentSchema(KafkaCluster cluster, Topic topic) {
         }
     }
 
-    record PersonBean(String firstName, String lastName, int age) {}
+    record PersonBean(String firstName, String lastName, int age) {
```

Comment:
How come these parameters aren't put on  separate lines?

Reply:
Guessing records are special. Will dig

Reply:
Yeah records are separated. but nothing I try will auto format as desired.

I can manually fix and they both leave it alone.